mods/common/chrome/editor.yaml

```diff
@@ -812,3 +812,28 @@ ScrollPanel@CATEGORY_FILTER_PANEL:
 			Height: 20
 			Visible: false
 
+ScrollPanel@COPY_FILTER_PANEL:
```

Comment:
Seems like this isn't just removing an empty newline?

Reply:
good catch, must have been a rebase error, I should rerun the rule